### PR TITLE
Proposed editorial changes

### PR DESCRIPTION
**Pull request by mihails-strasuns**
.

### CODE REVIEW

File: DIPs/DIP1004.md [hunk 1]
```diff
@@ -1,11 +1,11 @@
 # Inherited Constructors
 
-| Section         | Value                                                           |
-|-----------------|-----------------------------------------------------------------|
-| DIP:            | 1004                                                            |
-| Author:         | Andrej Mitrović                                                 |
-| Implementation: |                                                                 |
-| Status:         | Draft                                                           |
+| Section         | Value                                                      |
```
Comment (AndrejMitrovic) [inline DIPs/DIP1004.md:3]: Doesn't markdown already do line-wrapping when displayed on github?

**Comment by AndrejMitrovic** [comment]
This looks great, thanks a lot for the effort you put this in!

Should I do a squash-merge to my branch?

File: DIPs/DIP1004.md (hunk 1 above)
Comment (mihails-strasuns) [reply DIPs/DIP1004.md:3]: It does, but editing source becomes quite harder.

**Comment by mihails-strasuns** [comment]
> Should I do a squash-merge to my branch?

As you wish - I only have squash merges enabled for main DIP repo so in the end it will all look the same.

**Comment by mihails-strasuns** [comment]
Ping ? :)

**Comment by mihails-strasuns** [comment]
Ping @AndrejMitrovic , I think we are pretty good to go after this chunk of changes.

**Pull request merged**